client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -1,22 +1,27 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	config = require( 'config' ),
-	find = require( 'lodash/find' ),
-	includes = require( 'lodash/includes' );
+import React from 'react';
+import config from 'config';
+import find from 'lodash/find';
+import includes from 'lodash/includes';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-var Search = require( 'components/search' ),
-	UrlSearch = require( 'lib/mixins/url-search' ),
-	SectionNav = require( 'components/section-nav' ),
-	NavTabs = require( 'components/section-nav/tabs' ),
-	NavItem = require( 'components/section-nav/item' );
-
-let PeopleSearch = React.createClass( {
+import Search from 'components/search';
+import UrlSearch from 'lib/mixins/url-search';
+import SectionNav from 'components/section-nav';
+import NavTabs from 'components/section-nav/tabs';
+import NavItem from 'components/section-nav/item';
+import { localize } from 'i18n-calypso';
+import { getSelectedSite } from 'state/ui/selectors';
+import versionCompare from 'lib/version-compare';
+
+const PeopleSearch = React.createClass( { // eslint-disable-line react/prefer-es6-class
```

Comment:
I would suggest removing the disable eslint line. We do want to clean these up in the future